lib/jobFactory.js

```diff
@@ -46,8 +56,22 @@ class JobFactory extends BaseFactory {
 
         c.state = 'ENABLED';
         c.archived = false;
+        const perm = c.permutations[0];
+
+        return super.create(c).then((job) => {
+            if (!perm.annotations ||
+                !perm.annotations['beta.screwdriver.cd/buildPeriodically']) {
```

Comment:
We should keep it consistent with other features: https://docs.screwdriver.cd/user-guide/configuration/annotations

If we want to remove it, I would suggest to make an announcement that we are moving away from beta for ALL, not just this.

Reply:
Not necessarily, resources we are revisiting. If we had a common function to parse annotation name, then it would have been easier, since we are hardcoding annotation all over the place, change it now rather than later. 

I see no logic in calling this feature `beta` versus non `beta` This does not reflect in any way the `beta` status of screwdriver as a product.

Reply:
This brings my second ask, hide this behind a utility function, which should work if `beta` is present or not.

Reply:
The whole reason why we have `annotations` is because it's meant to be "experimental".
If we revisit, then we should revisit everything.